### DISCUSSION TITLE
\n code not executed

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#71] [General]
Into proposal screen, we can see on the description that is probably not executed some code but some text instead

![Screenshot 2023-05-24 at 19 34 16](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/744222fb-04ec-49cc-a660-65eff94d37d1)

**Comment by TJ-BitCanna** [comment]
Thanks for notifying! We'll dive into the markdown settings, although it might be difficult since part of the proposals are done on the V0.45.x SDK and have a different proposal formatting than the new format on the V0.46.x SDK. So it might be that we settle for the imperfect solution on this one.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- check if this is caused in the legacy governance module versus the v1

**Comment by TJ-BitCanna** [comment]
This is something we can't fix. Should be covered in the markdown configuration before sending a proposal on chain.
